sdk/cosmos/azure-cosmos/azure/cosmos/container.py

```diff
@@ -164,7 +181,6 @@ def read_item(
 
         :param item: The ID (name) or dict representing item to retrieve.
         :param partition_key: Partition key for the item to retrieve.
-        :param populate_query_metrics: Enable returning query metrics in response headers.
```

Comment:
Do we want to open an issue to track the full `overload` treatment for the rest of these methods?

Reply:
what do you mean by this?

Reply:
Just that it would be nice to add the overloads everywhere - but it's too much work to do right now, and non-breaking to add later. So we should open an issue to track it for post-GA.

Reply:
Follow-up issue to this: https://github.com/Azure/azure-sdk-for-python/issues/24423